modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -108,58 +164,168 @@ def show_found_exploits
 
     vprint_status "The following #{@local_exploits.length} exploit checks are being tried:"
     @local_exploits.each do |x|
-      vprint_status x.fullname
+      vprint_status x[:module].fullname
     end
   end
 
   def run
-    if @local_exploits.empty?
+    wanted_modules = @local_exploits.select { |mod| mod[:wanted] }
+    if wanted_modules.empty?
       print_error 'No suggestions available.'
+      vprint_line
+      vprint_session_info
+      vprint_status unwanted_modules_table(@local_exploits.reject{ |mod| mod[:wanted] })
```

Comment:
I think in the old code `module_wanted?` was a nice name as a filter, i.e. loop through the modules and filter the unwanted modules.

But I think in this new iteration of code `unwanted_modules` isn't very clear on what that actually means 🤔
I wonder if there a better name that could be used here? Is this really `incompatible_modules` and `compatible_modules` ?